shuup/default_tax/models.py

```diff
@@ -17,11 +17,12 @@
 
 class TaxRuleQuerySet(models.QuerySet):
     def may_match_postal_code(self, postalcode):
-        null = Q(_postal_codes_min__isnull=True)
+        null = Q(Q(_postal_codes_min__isnull=True) | Q(_postal_codes_min=""))
+        star = Q(_postal_codes_min="*")
         in_range = Q()
         if postalcode:
             in_range = Q(_postal_codes_min__lte=postalcode, _postal_codes_max__gte=postalcode)
-        return self.filter(null | in_range)
+        return self.filter(star | null | in_range)
```

Comment:
I think this should be fixed by making Pattern.get_alphabetical_limits() return (None, None) for *. Then there would be no need to check * pattern here.